01-atoms/text/Paragraph.component.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import withModifiers from '../../_utils/withModifiers';
+
+const Paragraph = ({ children, modifiers }) => {
+  return <p className={withModifiers('paragraph')(modifiers)}>{children}</p>;
+};
+
+Paragraph.propTypes = {
+  modifiers: PropTypes.arrayOf(PropTypes.string),
```

Comment:
I see this shape now appearing 5 times in 4 components
`PropTypes.arrayOf(PropTypes.string)`
While this is maybe a trivial example, whenever I see duplicates, I have the urge to refactor and declare it once, so all instances stay in sync. I wonder if we can create a `modifiersPropType` variable? 

Just something to think about. I've been working in Flow so I don't know if there is a specific way to customize these PropTypes.

Reply:
I'd argue these are pretty simple, even if duplicated a bit, each one defines the props for each. I don't think we're ready for unifying these yet.